presentation/src/main/java/me/androidbox/presentation/agenda/screen/AgendaScreen.kt

```diff
@@ -92,6 +100,19 @@ fun AgendaScreen(
             agendaScreenEvent(AgendaScreenEvent.OnDateChanged(localDate))
         }
     )
+
+    AgendaDropDownMenu(
+        modifier = Modifier.background(color = MaterialTheme.colorScheme.dropDownMenuBackgroundColor),
+        shouldOpen = agendaScreenState.agendaDropdownItemId.isNotEmpty(),
```

Comment:
When the state has a list of dropdown items (string resources) then show the dropdownmenu.

Reply:
@MKiperszmid Updated